File: TodayFitComplete_Backend/src/main/java/com/SJY/TodayFitComplete_Backend/dto/request/board/BoardWriteDto.java
package com.SJY.TodayFitComplete_Backend.dto.request.board;

import com.SJY.TodayFitComplete_Backend.entity.Board;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

/**
 * 새 게시글 등록을 위한 요청 DTO.
 */
@Getter
@Setter
@NoArgsConstructor
public class BoardWriteDto {
    private String title;
    private String content;

    public static Board ofEntity(BoardWriteDto dto) {
        return Board.builder()
                .title(dto.getTitle())
                .content(dto.getContent())
                .build();
    }
}

File: TodayFitComplete_Frontend/app/src/main/java/kr/ac/kumoh/ce/s20190633/todayfitcomplete_frontend/Dto/Board/BoardWriteDto.kt
package kr.ac.kumoh.ce.s20190633.todayfitcomplete_frontend.Dto.Board

// BoardWriteDto 클래스: 게시글을 작성할 때 필요한 데이터를 나타냅니다.
data class BoardWriteDto(
    val title: String,                   // 게시글 제목
    val content: String                  // 게시글 내용
)
